### ISSUE TITLE
arria-errormessage and aria-description are missing at 4.8.1

### ISSUE THREAD

**Issue by JAWS-test** [w3c/core-aam#83]
I suspect that in chapter ["4.8.1 State and Property Change Events"](https://w3c.github.io/core-aam/#mapping_events_state-change) [aria-errormessage](https://rawgit.com/w3c/aria/master/#aria-errormessage) and [aria-description](https://rawgit.com/w3c/aria/master/#aria-description) are missing, because both behave like aria-describedby and aria-describedby is mentioned in the chapter
